apps/auth/ssr/src/app/page.tsx

```diff
@@ -1,22 +1,48 @@
+import { cookies } from "next/headers";
 import { createCrossmint, CrossmintAuth } from "@crossmint/server-sdk";
 
-export default function Home() {
+export default async function Home() {
+    let isLoggedIn = false;
+    let userId: string | undefined;
+
     try {
         const crossmint = createCrossmint({
             apiKey: process.env.SERVER_CROSSMINT_API_KEY || "",
         });
         const crossmintAuth = CrossmintAuth.from(crossmint);
```

Comment:
re: `.from` instantiation
It makes sense to have this as a static function because maybe one day we need to make it async that a normal constructor can't handle, so maybe we at least just make it `init` to match smart wallets
<img width="823" alt="image" src="https://github.com/user-attachments/assets/815efecf-9345-4379-91d7-e9b7500fab10">

Reply:
we discussed that this was an old approach and we should go with `from`